float64 command
float64 command_dot
---
float64 command
float64 command_dot
